Assets/Scripts/Data/PickupPup/Currency/DogFoodDataS.cs

```diff
@@ -0,0 +1,32 @@
+﻿/*
```

Comment:
I think it's confusing to inherit from `CurrencyData` instead of either subclassing `DogFoodData` or creating a variable inside of `DogFoodData` which indicates the type of food.